### PR TITLE
Export functions individually

### PR DESCRIPTION
**Pull request by voxpelli**
This enables one to do:

```js
import { getMessage } from 'check-error';
```

Previous code required:

```js
import checkError from 'check-error';
const { getMessage } = checkError;
```

### CODE REVIEW

**Review by keithamus — APPROVED** [review]

**Comment by keithamus** [comment]
@koddsson would you like to land this one?

**Comment by voxpelli** [comment]
Do btw note that this would be a breaking change, requiring to be released as eg 3.0.0

**Comment by keithamus** [comment]
We could always keep both and not break 😃

File: index.js [hunk 1]
```diff
@@ -121,5 +121,3 @@ function getMessage(errorLike) {
 
   return msg;
 }
-
-export { compatibleInstance, compatibleConstructor, compatibleMessage, getMessage, getConstructorName };
```
Comment (koddsson) [inline index.js:125]: Yeah, what do you think about keeping this, and we can release this as 3.0.1?

**Comment by voxpelli** [comment]
Never mind this PR, one shouldn't do PR:s at late Friday evenings evidently.

I mixed up `export { ... };` with `export default { ... };`, the latter matching the behavior of a CJS module whereas the former matches individual exports.

So: My PR is functionally identical to the existing code and as such a simple case of code formatting preference and as such a rather needless one.

**Pull request closed without merging**